src/search/landmarks/landmark_count_heuristic.cc

```diff
@@ -374,8 +421,21 @@ static shared_ptr<Heuristic> _parse(OptionParser &parser) {
                             "(see OptionCaveats#Using_preferred_operators_"
                             "with_the_lmcount_heuristic)", "false");
     parser.add_option<bool>("alm", "use action landmarks", "true");
+    parser.add_option<int>("derived_lm_cost",
+                           "the cost to use for derived landmarks",
+                           "1", Bounds("0", "1"));
```

Comment:
What's the behaviour if I call `lmcount` with `admissible=true` and `derived_lm_cost=1`? Is it still admissible? I didn't follow the discussions in the issue tracker and on Discord too closely, so you probably have explained it there already.
Personally, I think the default "0" is more intuitive and in my opinion it wouldn't hurt to make the alias explicitly specify that we use derived landmark costs "1", but I can live with either option.

Reply:
(for the first part, see my comment above)
I was on the fence whether 0 or 1 should be the default and decided for 1 because it performs better empirically. I also hope that once we get around to thinking about how to deal with derived landmarks properly we can get rid of the option again.

Reply:
As mentioned on Discord: leaving 1 as default because it's the better performing option is reasonable. I suggest to add a comment to explain why this default performs better.